test/apply-command.test.js

```diff
@@ -6,6 +6,32 @@ const spawn = require('cross-spawn')
 const FILES_PATH = path.join(__dirname, 'files-to-mod')
 const PATH_ACTUAL = path.join(FILES_PATH, 'actual-files')
 
+const apply = options => {
+    const result = spawn.sync(
+        './bin/d2-utils-codemods',
+        ['apply', ...options],
+        {
+            env: {
+                ...process.env,
+                RESOLVE_PATH: path.join(__dirname, 'node_modules'),
+            },
+        }
+    )
+
+    if (result.error) {
+        console.error(result.error)
+    }
+
+    if (result.stderr && result.stderr.toString()) {
+        result.strout && console.log('STDOUT:')
+        result.strout && console.log(result.stdout.toString())
+        console.log('STDERR:')
+        console.log(result.stderr.toString())
```

Comment:
Think I mentioned this before... is there a reason not to use `exec` from `@dhis2/cli-helpers-engine` here?  It already supports piping stdout and stderr to the parent (this dumps stderr to stdout which seems incorrect?). If we want to use `cross-spawn` we should update the helper library instead

Reply:
Ah sorry I see this is a test.... nevermind - maybe we should snapshot stdout and stderr rather than just dumping it to console.log though?

Reply:
See comment below